config/cruisecontrol.properties

```diff
@@ -206,17 +206,23 @@ goal.violation.exclude.recently.removed.brokers=true
 # The zk path to store failed broker information.
 failed.brokers.zk.path=/CruiseControlBrokerList
 
-# The maximum time in milliseconds to store the response and access details of a completed user task.
-completed.user.task.retention.time.ms=86400000
+# The maximum time in milliseconds to store the response and access details of a completed monitoring user task.
+completed.monitor.user.task.retention.time.ms=86400000
+
+# The maximum time in milliseconds to store the response and access details of a completed admin user task.
+completed.admin.user.task.retention.time.ms=86400000
 
 # The maximum time in milliseconds to retain the demotion history of brokers.
 demotion.history.retention.time.ms=1209600000
 
 # The maximum time in milliseconds to retain the removal history of brokers.
 removal.history.retention.time.ms=1209600000
 
-# The maximum number of completed user tasks for which the response and access details will be cached.
-max.cached.completed.user.tasks=100
+# The maximum number of completed monitoring user tasks for which the response and access details will be cached.
+max.cached.completed.monitor.user.tasks=40
+
+# The maximum number of completed admin user tasks for which the response and access details will be cached.
+max.cached.completed.admin.user.tasks=60
```

Comment:
Can we please refrain from changing existing configs (because they are breaking changes) or their values (because this patch is about `Sematic categorization of user tasks`)?

Reply:
Yeah I'll keep the old configs.
After adding semantic categorization, we will no longer use the old retention time or limit as specified by `max.cached.completed.user.tasks` and `http://completed.user.task.retention.time.ms`, as end category will have its custom retention count and time. Do you see the configs still play a role in any way or do we keep them for compatibility?